stem/water_boundaries.py

```diff
@@ -0,0 +1,105 @@
+from typing import List, Union
+from dataclasses import dataclass, field
+from abc import ABC
+
+
+
+@dataclass
+class WaterBoundaryParameters(ABC):
+    """
+    Abstract base class for load water boundary parameters
+
+    Attributes:
+        - surfaces_assigment (List[str]): List of surfaces to which the water boundary is assigned.
+        - is_fixed (bool): True if the water boundary is fixed, False otherwise.
+        - gravity_direction (int): Direction of the gravity vector.
+        - out_of_plane_direction (int): Direction of the out of plane vector.
+
+
+    """
+    surfaces_assigment: List[str] = field(default_factory=lambda: [""])
+    is_fixed: bool = True
```

Comment:
these parameters should not be given a default. As wrong stuff can happen in 3D situations, when the user keeps it on the default values